app/src/main/java/com/example/next_show/fragments/FeedFragment.java

```diff
@@ -157,4 +175,46 @@ public void navigate(View v, Bundle b){
             Navigation.findNavController(v).navigate(R.id.action_feedFragment_to_showDetailFragment, b);
         }
     }
+
+    class RelatedCallback implements ResponseCallback {
+        @Override
+        public void onSuccess(List<com.uwetrottmann.trakt5.entities.Show> shows) {
+            // turn all of these into usable Show objects
+            List<Show> updatedShows = Show.fromRecShows(shows);
```

Comment:
You do this in all of your response callbacks -- I would update the on success to just pass your list of shows?  Is there a use case, where you don't need to do this `Show.fromRecShows(shows)` or populate your show objects in a different way?